video_utils.py

```
@@ -534,9 +566,7 @@ async def sample_frames_equidistant(
             ]
 
             def _run_one():
-                return subprocess.run(
-                    cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, check=False
-                )
+                return _safe_subprocess_run(cmd)
```

Ruff (0.14.10)

Function definition does not bind loop variable `cmd`

(B023)